external/fv3fit/tests/training/test_main.py

```diff
@@ -32,8 +32,8 @@ def get_mock_dataset(n_time, unstacked_dims: Sequence[str]):
             "downward_shortwave": (dims_surface, arr_surface),
             "net_shortwave": (dims_surface, arr_surface),
             "downward_longwave": (dims_surface, arr_surface),
-            "dQ1": (dims, arr),
-            "dQ2": (dims, arr),
+            "Q1": (dims, arr),
+            "Q2": (dims, arr),
```

Comment:
it seems a bit outside the scope of this PR to add the precipitative model test (it fails even if I undo the changes I made to `test_main.py`). But I will parameterize some of the output variable/transform changes I made, so that it will be possible to run the precipitative model test without those.

Reply:
OK, thanks for trying. I was hoping it would pass on main.